In [1]:
# Import Packages
import pandas as pd
import numpy as np

In [3]:
# Load Dataset
objects = r"C:\Users\Yusuf Saputra\OneDrive - University of Toronto\Documents\UofT\Winter 2024\ECO225\ECO225 Project\Data\startup\objects.csv"
df_objects = pd.read_csv(objects)

df_objects.head()

C:\Users\Yusuf Saputra\AppData\Local\Temp\ipykernel_9276\1154733044.py:3: DtypeWarning: Columns (3,7,9,10,17,18,21,22,23,25,26,29,30,37) have mixed types. Specify dtype option on import or set low_memory=False.
  df_objects = pd.read_csv(objects)


id entity_type  entity_id parent_id                name  \
0      c:1     Company          1       NaN            Wetpaint   
1     c:10     Company         10       NaN             Flektor   
2    c:100     Company        100       NaN               There   
3  c:10000     Company      10000       NaN             MYWEBBO   
4  c:10001     Company      10001       NaN  THE Movie Streamer   

      normalized_name                    permalink    category_code  \
0            wetpaint            /company/wetpaint              web   
1             flektor             /company/flektor      games_video   
2               there               /company/there      games_video   
3             mywebbo             /company/mywebbo  network_hosting   
4  the movie streamer  /company/the-movie-streamer      games_video   

      status  founded_at  ... last_funding_at funding_rounds  \
0  operating  2005-10-17  ...      2008-05-19              3   
1   acquired         NaN  ...             NaN              0   
2   acquired         NaN  ...             NaN              0   
3  operating  2008-07-26  ...             NaN              0   
4  operating  2008-07-26  ...             NaN              0   

  funding_total_usd first_milestone_at last_milestone_at  milestones  \
0        39750000.0         2010-09-05        2013-09-18           5   
1               0.0                NaN               NaN           0   
2               0.0         2003-02-01        2011-09-23           4   
3               0.0                NaN               NaN           0   
4               0.0                NaN               NaN           0   

   relationships        created_by           created_at           updated_at  
0             17  initial-importer  2007-05-25 06:51:27  2013-04-13 03:29:00  
1              6  initial-importer  2007-05-31 21:11:51  2008-05-23 23:23:14  
2             12  initial-importer  2007-08-06 23:52:45  2013-11-04 02:09:48  
3              0               NaN  2008-08-24 16:51:57  2008-09-06 14:19:18  
4              0               NaN  2008-08-24 17:10:34  2008-09-06 14:19:18  

[5 rows x 40 columns]

In [7]:
df_objects.shape

(462651, 40)

This tells us that the dataset that we choose contains 462,651 rows x 40 columns.